examples/key_value_store.rs

```diff
@@ -38,12 +38,14 @@ extern crate maidsafe_utilities;
 #[macro_use]
 extern crate unwrap;
 extern crate docopt;
-extern crate rust_sodium;
 #[macro_use]
 extern crate serde_derive;
+#[cfg(not(feature = "use-mock-crust"))]
+extern crate sha2;
```

Comment:
1. Why are we doing this ? Does not safe_crypto provide this for both mock and non-mock ?
2. We don't use sha2 anyway having migrated to sha3